manifests/o/OpenArena/OpenArena/0.8.8/OpenArena.OpenArena.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with komac v2.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: OpenArena.OpenArena
+PackageVersion: 0.8.8
+InstallerType: zip
+NestedInstallerType: portable
+ArchiveBinariesDependOnPath: true
+NestedInstallerFiles:
+  - RelativeFilePath: openarena-0.8.8/openarena.exe
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2018-02-05
+Installers:
+  - Architecture: neutral
+    InstallerUrl: https://www.moddb.com/downloads/start/40515
```

Try using the Sourceforge URL [suggested](https://github.com/microsoft/winget-pkgs/issues/111294#issuecomment-2645858316) in the original issue.

---

Oh, sorry. I did not notice that the "Error-Hash-Mismatch" label was already removed.

---

```suggestion
    InstallerUrl: https://sourceforge.net/projects/oarena/files/openarena-0.8.8.zip
```

Switch to _Source Forge_ URL.